elasticsearch/config/elasticsearch.yml

```diff
@@ -13,4 +13,4 @@ discovery.zen.minimum_master_nodes: 1
 ## Use single node discovery in order to disable production mode and avoid bootstrap checks
 ## see https://www.elastic.co/guide/en/elasticsearch/reference/current/bootstrap-checks.html
 #
-discovery.type: single-node
+discovery.type: zen
```

Comment:
@antoineco Do you think that might introduce issues with the standard deployment method (e.g. compose)?

Reply:
Sorry that's a leftover from my local experiments.  I'll remove that.